schedule.go

```diff
@@ -0,0 +1,187 @@
+package helix
+
+// GetScheduleParams are the parameters for GetSchedule
+type GetScheduleParams struct {
+	BroadcasterID string `json:"broadcaster_id"`
+	ID            string `json:"id"`
+	StartTime     string `json:"start_time"`
+	UTCOffset     string `json:"utc_offset"`
+	First         int    `json:"first"`
+	After         string `json:"after"`
+}
+
+// GetScheduleResponse is the response data in GetSchedule
+type GetScheduleResponse struct {
+	ResponseCommon
+
+	Data GetScheduleData
+}
+
+type GetScheduleData struct {
+	Schedule   ScheduleData          `json:"data"`
+	Pagination GetSchedulePagination `json:"pagination"`
+}
+
+type ScheduleData struct {
+	Segments         []GetScheduleSegment `json:"segments"`
+	BroadcasterID    string               `json:"broadcaster_id"`
+	BroadcasterName  string               `json:"broadcaster_name"`
+	BroadcasterLogin string               `json:"broadcaster_login"`
+	Vacation         GetScheduleVacation  `json:"vacation"`
+}
+
+type GetScheduleSegment struct {
+	ID            string                     `json:"id"`
+	StartTime     string                     `json:"start_time"`
+	EndTime       string                     `json:"end_time"`
+	Title         string                     `json:"title"`
+	CanceledUntil string                     `json:"canceled_until"`
+	Category      GetScheduleSegmentCategory `json:"category"`
+	IsRecurring   bool                       `json:"is_recurring"`
+}
+
+type GetScheduleSegmentCategory struct {
+	ID   string `json:"id"`
+	Name string `json:"name"`
+}
+
+type GetScheduleVacation struct {
+	StartTime string `json:"start_time"`
+	EndTime   string `json:"end_time"`
```

Comment:
`StartTime` & `EndTime` should be of type [`Time`](https://github.com/nicklaw5/helix/blob/main/time.go#L24), rather than `string`.